en/products/relay/bundle.ftl

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/relay/
+
+# Variables:
+#   $monthly_price (string) - the monthly cost (including currency symbol) for a given plan. Examples: $0.99, 0,99 €
+bundle-banner-header-2 = Limited-time only: { -brand-name-relay-premium } + { -brand-name-mozilla-vpn } for { $monthly_price }/month
+# Variables:
+#   $savings (string) - the percentage saved (including % symbol) for a given plan. Examples: 50%, 70%
+bundle-banner-body-3 = { -brand-name-mozilla-vpn } protects you from tracking & surveillance while you’re gaming, online banking, or getting work done. Add it to { -brand-name-firefox-relay } for { $savings } off.
+bundle-banner-plan-header-2 = Your 1-year plan includes:
+bundle-banner-plan-modules-email-masking = Email masking
+bundle-banner-plan-modules-phone-masking = Phone masking
+bundle-banner-plan-modules-mozilla-vpn = { -brand-name-mozilla-vpn }
+bundle-banner-cta = Get { -brand-name-mozilla-vpn } + { -brand-name-relay }
+# Variables:
+#   $days_guarantee (string) - the number of days for money-back guarantee. Examples: 30, 90
+bundle-banner-money-back-guarantee-2 = { $days_guarantee }-day money back guarantee for first time subscribers
+# Variables:
+#   $num_vpn_servers (string) - the number of VPN servers. Examples: 400, 500, 600
+bundle-feature-one = More than { $num_vpn_servers } servers
```

Comment:
Since we haven't translated this yet, we should be able to make the change without versioning the ID?

Reply:
I suggested the change so that translations wouldn't get migrated - that being said I do have the ability to ignore specific strings, so in this case we don't have to change the ID. (I'll just need to update my automated shell script to include this parameter.

Reply:
I changed this string to `bundle-feature-more-than-servers` which changes the string ID without adding a version number. I did this with a few other strings that needed versioning too but it didn't make sense in all cases.